src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -98,8 +100,9 @@ private static Client createEditedClient(Client clientToEdit, EditClientDescript
         Email updatedEmail = editClientDescriptor.getEmail().orElse(clientToEdit.getEmail());
         Address updatedAddress = editClientDescriptor.getAddress().orElse(clientToEdit.getAddress());
         Set<Tag> updatedTags = editClientDescriptor.getTags().orElse(clientToEdit.getTags());
+        List<Procedure> updatedProcedures = editClientDescriptor.getProcedures().orElse(clientToEdit.getProcedures());
```

Comment:
Hi JB, thank you for the question! :) We have a separate method for editing the Client and the Procedure, but since the Procedure is tagged to the Client, In editting the Client, I would have to save the Procedures as well. This method would not allow editting for any Procedures. I have since editted the code to make it clearer :)